mage_ai/orchestration/db/__init__.py

```diff
@@ -3,6 +3,7 @@
 from urllib.parse import parse_qs, quote_plus, urlparse
 
 import sqlalchemy
+from opentelemetry.instrumentation.sqlalchemy import SQLAlchemyInstrumentor
```

Comment:
Let's only import and use it when OTEL feature is enabled. Then opentelemetry libraries are not installed in the base Mage package.